QuickTCGA/Logic/TCGAUtilities.h

```diff
@@ -23,6 +23,8 @@
 
 #include <csignal>
 
+#include "vtkSlicerQuickTCGAModuleLogicExport.h"
```

Comment:
Thanks for the clarification, I thought it is a standard file, but wanted to confirm.